Xamarin.PropertyEditing/ViewModels/PropertyGroupViewModel.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Xamarin.PropertyEditing.ViewModels
+{
+	internal class PropertyGroupViewModel
+		: EditorViewModel, IFilterable
+	{
+		public PropertyGroupViewModel (string category, IEnumerable<PropertyViewModel> properties, IEnumerable<IObjectEditor> objEditors)
+			: base (objEditors)
+		{
+			if (category == null)
+				throw new ArgumentNullException (nameof(category));
+			if (properties == null)
+				throw new ArgumentNullException (nameof (properties));
+
+			Category = category;
+
+			this.properties = properties.ToArray ();
+			foreach (var vm in this.properties) {
+				if (vm.IsAvailable)
+					this.propertiesView.Add (vm);
+
+				vm.PropertyChanged += OnChildPropertyChanged;
+			}
+		}
+
+		public IReadOnlyList<PropertyViewModel> Properties => this.propertiesView;
+
+		public override string Name
```

Comment:
Shouldn't this be initialized or have an actual getter?

Reply:
No, property groups don't have names.

Reply:
I'd be explicit and return null then.